src/license_text_normalizer/main.py

```diff
@@ -147,7 +148,8 @@ def _strip_words(line: str, words_to_strip: List[str]) -> str:
     return normalized_line.lstrip()
 
 
-def _is_repeated_non_alnum(line: str) -> bool:
-    if not len(line) > 1:
+def _is_line_non_alnum(line: str, delimiters: List[str]) -> bool:
+    if not (len(line) > 1 and line[0] in delimiters):
         return False
-    return not line[0].isalnum() and line == len(line) * line[0]
+    line_without_spaces = line.replace(" ", "")
+    return not line_without_spaces.isalnum()
```

Comment:
I think we'll need to check character-by-character rather than at the string level. `.isalnum()` will only return `True` if *all* of the characters are alphanumeric, but what we want to do is to not strip lines that contain *any* alphanumeric characters.

If there's not a built-in function, then I think something like this would work:
```
return not any(char.isalnum() for char in line)
```